### PR TITLE
[Brave News]: Add translations for new channels

### PR DESCRIPTION
**Pull request by fallaciousreasoning**
Resolves https://github.com/brave/brave-browser/issues/27024

## Submitter Checklist:

- [ ] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

1. Either run Brave with `--brave-today-host=brave-today-cdn.bravesoftware.com` or normally, once the staging backend has been deployed.
2. Open the Brave News customize dialog
3. Check the dev console - there should be no `Couldn't find translation for channel` messages

### CODE REVIEW

File: browser/ui/webui/brave_webui_source.cc [hunk 1]
```diff
@@ -233,19 +233,26 @@ void CustomizeWebUIHTMLSource(content::WebUI* web_ui,
         { "braveNewsChannel-Crypto", IDS_BRAVE_NEWS_CHANNEL_CRYPTO},
         { "braveNewsChannel-Culture", IDS_BRAVE_NEWS_CHANNEL_CULTURE},
         { "braveNewsChannel-Entertainment", IDS_BRAVE_NEWS_CHANNEL_ENTERTAINMENT}, // NOLINT
+        { "braveNewsChannel-Entertainment News", IDS_BRAVE_NEWS_CHANNEL_ENTERTAINMENT_NEWS}, // NOLINT
```
Comment (simonhong) [inline browser/ui/webui/brave_webui_source.cc:236]: Curious the strings that have space works?

**Review by simonhong — APPROVED** [review]
++per CI all green.

File: browser/ui/webui/brave_webui_source.cc (hunk 1 above)
Comment (fallaciousreasoning) [reply browser/ui/webui/brave_webui_source.cc:236]: It's just a key in a JavaScript object :)

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

Brave | 1.48.32 Chromium: 108.0.5359.94 (Build oficial) nightly (x86_64)
-- | --
Revisión | 713576b895246504ccc6b92c2fb8ce2d60194074-refs/branch-heads/5359_71@{#3}
SO | macOS Versión 11.7.1 (Fase 20G918)

### Steps:
1. installed `1.48.32`
2. set my macOS language to `Espanol`
3. launched Brave
4. opened a new-tab page
5. opted in to Brave News
6. clicked on `Personlizar`
7. looked at the channel names

#### Confirmed the *start* of translations for the newly-added strings: Tiempo (Weather), Videojuegos (Gaming), Noticias internacionales (World News), etc.

example | example
---------|---------
Captura de Pantalla 2022-12-07 a la(s) 10 52 43 a  m | Captura de Pantalla 2022-12-07 a la(s) 10 51 44 a  m

**Pull request merged**